client/components/wcc-settings-form/index.js

```diff
@@ -8,43 +8,41 @@ import * as FormActions from 'state/form/actions';
 import { successNotice, errorNotice } from 'state/notices/actions';
 import * as FormValueActions from 'state/form/values/actions';
 import * as PackagesActions from 'state/form/packages/actions';
-import { getFormErrors, getStepFormErrors } from 'state/selectors';
+import { getFormErrors, getStepFormErrors, getStepFormSuggestions } from 'state/selectors/errors';
 import { translate as __ } from 'lib/mixins/i18n';
 import ActionButtons from 'components/action-buttons';
+import Tabs from 'components/tabs';
+import isEmpty from 'lodash/isEmpty';
 
 const WCCSettingsForm = ( props ) => {
 	const currentStep = props.form.currentStep;
-	const currentStepLayout = props.layout[ currentStep ];
 
-	const renderCurrentStep = () => {
-		if ( ! currentStepLayout ) {
-			return <span>... spinner or something ...</span>;
+	const renderGroup = ( stepIndex ) => {
+		if ( ! props.layout[ stepIndex ] ) {
+			return null; // TODO: display a spinner here
 		}
 
 		return (
 			<WCCSettingsGroup
 				{ ...props }
-				group={ currentStepLayout }
+				group={ props.layout[ stepIndex ] }
 				saveForm={ props.formValueActions.submit }
+				key={ stepIndex }
 			/>
 		);
 	};
 
-	const renderTab = ( label, index ) => {
-		if ( index === currentStep ) {
-			return <b>{ label }</b>;
-		} else if ( index < currentStep ) {
-			return <a onClick={ () => props.formActions.goToStep( index ) }>{ label }</a>;
-		}
-		return <span>{ label }</span>;
-	};
-
-	const renderMultiStepForm = () => {
+	const getActionButtons = () => {
 		const buttons = [];
+		const currentStepLayout = props.layout[ currentStep ];
+		let submitEnabled = currentStepLayout && isEmpty( props.stepErrors ) && ! props.form.isSaving;
+		if ( ! isEmpty( props.stepSuggestions ) ) {
```

Comment:
Please add a comment here explaining why we want to enable submit in this case to aid the next person editing this logic.